lib/OnyxUtils.ts

```diff
@@ -1240,12 +1269,14 @@ function updateSnapshots(data: OnyxUpdate[], mergeFn: typeof Onyx.merge): Array<
  * @param collection Object collection keyed by individual collection member keys and values
  * @param mergeReplaceNullPatches Record where the key is a collection member key and the value is a list of
  * tuples that we'll use to replace the nested objects of that collection member record with something else.
+ * @param retryAttempt retry attempt
  */
 function mergeCollectionWithPatches<TKey extends CollectionKeyBase, TMap>(
```

Comment:
I think at this point, we could start thinking about changing the function signature to receive one object with named property parameters. To prevent calls like this [in line 1384](https://github.com/Expensify/react-native-onyx/pull/694/files#diff-c584aabafbadccee45af6a779a6e26057cf5b2546ed0de9d69a8bc9ac1bae791R1384) where we have to explicitly pass `undefined` or the default value:

`mergeCollectionWithPatches, [collectionKey, resultCollection, undefined, false], retryAttempt)`